core/services/feeds/orm.go

```diff
@@ -113,6 +115,27 @@ WHERE id = ?;
 	return &mgr, nil
 }
 
+// GetManagers gets feeds managers by ids
+func (o *orm) GetManagers(ctx context.Context, ids []int64) (managers []FeedsManager, err error) {
+	stmt := `
+SELECT id, name, uri, public_key, job_types, is_ocr_bootstrap_peer, ocr_bootstrap_peer_multiaddr, created_at, updated_at
```

Comment:
I'm against using `SELECT *` because it can lead to run time errors if you add a new column but don't add the field to the model. I prefer the explicitness of this, although we do use `SELECT *` in other places